mobile/apps/photos/lib/utils/image_ml_util.dart

```diff
@@ -65,6 +65,8 @@ Future<DecodedImage> decodeImageFromPath(
           await FlutterImageCompress.compressWithFile(
         imagePath,
         format: CompressFormat.jpeg,
+        minWidth: 8000, // High value to ensure image is not scaled down
+        minHeight: 8000, // High value to ensure image is not scaled down
```

Comment:
Should we keep the limit higher?
Also, let's test and validate that this is working as expected?
Ideally, if the decodedImage width or high is matching the higher limit, we should probably fail and log the values.